packages/mui-base/src/ButtonUnstyled/ButtonUnstyled.tsx

```diff
@@ -58,10 +61,13 @@ const ButtonUnstyled = React.forwardRef(function ButtonUnstyled<
   } = props;
 
   const buttonRef = React.useRef<HTMLButtonElement | HTMLAnchorElement | HTMLElement>();
-  const handleRef = useForkRef(buttonRef, ref);
+  const handleRef = useForkRef(buttonRef, forwardedRef);
+
+  const ButtonRoot: React.ElementType = component ?? components.Root ?? 'button';
```

Comment:
The `components` prop was removed, but we are still handling it. Something is off here, or I missed something.

Reply:
Ah, my bad I confused it with the `ButtonUnstyled`, thanks for the clarification.

Reply:
Looks like it was removed from both:

![image](https://user-images.githubusercontent.com/4512430/155699325-71dd8a7b-8f2d-4d34-b5d3-75b20592aab2.png)

Or something is wrong with the API generation